drinks_touch/game.py

```diff
@@ -99,6 +102,11 @@ def main(argv):
 
     init_db()
 
+    # admin = KeycloakAdmin()
+    # users = admin.get_user_list()
+    # from pprint import pprint
+    # pprint(users)
+
```

Comment:
Can be removed? Or still useful for some testing?

Reply:
Could serve as a reminder on how to use the class, but we could certainly move it somewhere else. Maybe into the oidc.py file? Next steps would anyway be to move loading the users directly from ldap to instead load from the database, so right now this code isn't too relevant. I'll move it. Any other suggestion where to put it so we can find it later?